client/blocks/signup-form/utils.jsx

```diff
@@ -0,0 +1,98 @@
+export const suggestCorrectEmail = ( inputDomain ) => {
+	const validDomains = [
+		'gmail.com',
+		'yahoo.com',
+		'hotmail.com',
+		'aol.com',
+		'outlook.com',
+		'comcast.net',
+		'icloud.com',
+		'msn.com',
+		'hotmail.co.uk',
+		'sbcglobal.net',
+	];
+
+	const extractDomainWithExtension = ( email ) => {
+		const atIndex = email.indexOf( '@' );
+		if ( atIndex !== -1 ) {
+			return email.slice( atIndex + 1 );
```

Comment:
Maybe lowercase it too?

Reply:
Good catch, lets make sure to test a capitalized email.